apps/cms/src/api/test/content-types/test/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "kind": "collectionType",
+  "collectionName": "tests",
+  "info": {
+    "singularName": "test",
+    "pluralName": "tests",
+    "displayName": "Test",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "title": {
+      "type": "string",
+      "required": true,
+      "unique": true
+    },
+    "passingScore": {
+      "type": "decimal",
+      "required": true,
+      "min": 0,
+      "max": 100,
+      "default": 0
+    },
+    "status": {
```

Comment:
We have draftAndPublish option model 
https://docs.strapi.io/dev-docs/backend-customization/models#model-options
https://docs.strapi.io/user-docs/content-manager/saving-and-publishing-content#publishing-a-draft